pkg/tracker/index.go

```diff
@@ -160,27 +172,27 @@ type IndexTracker struct {
 }
 
 type DataSource interface {
-	Get() ([]byte, error)
+	Get(ctx context.Context) ([]byte, error)
```

Comment:
On a second thought I now see that you use the ctx to get a client instance but we removed this pattern and now don't use the context for storing these so this param can be removed.

```
u.client = ctx.Value(tellorCommon.ClientContextKey).(rpc.ETHClient)
```
Instead of this need to pass it in the `NewUniswap` constructor. See the pr for removing the global context.